### PR TITLE
Irminger 9 and Pioneer 18c cruiseinfo updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
- Updated Irminger 9 start/end dates/times

- Added Pioneer 18c glider cruise

From Sheri on Slack #cgsn-data-team - Irminger 9:
The Irminger 9 cruise departed WHOI at 16:00 local (20:00 UTC) on 20 June 2022.
The Irminger 9 Cruise ended at 08:35 UTC (and Reykjavik local) on 16 July 2022.

From Collin on Slack #cgsn-data-team - P18c (also can be found in QLR 3204-01810):
TS20220714,Tenacious II,2022-07-14T02:00:00,2022-07-15T04:00:00,Pioneer 18c glider deployments

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -286,8 +286,9 @@ TS20220518,Tenacious II,2022-05-18T11:00:00,2022-05-19T03:00:00,Pioneer 18a glid
 TS20220526,Tenacious II,2022-05-26T01:00:00,2022-05-26T23:00:00,Pioneer 18b test glider deployments and cp_379 recovery
 EK20220526,Elakha,2022-05-26T16:02:00,2022-05-26T22:22:00,Endurance CE02SHSP turn
 EK20220617,Elakha,2022-06-17T17:50:00,2022-06-17T23:00:00,Endurance glider deployments
-AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
+AR69-01,Neil Armstrong,2022-06-20T20:00:00,2022-07-16T08:35:00,Irminger 9
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:289]: Looks good

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -286,8 +286,9 @@ TS20220518,Tenacious II,2022-05-18T11:00:00,2022-05-19T03:00:00,Pioneer 18a glid
 TS20220526,Tenacious II,2022-05-26T01:00:00,2022-05-26T23:00:00,Pioneer 18b test glider deployments and cp_379 recovery
 EK20220526,Elakha,2022-05-26T16:02:00,2022-05-26T22:22:00,Endurance CE02SHSP turn
 EK20220617,Elakha,2022-06-17T17:50:00,2022-06-17T23:00:00,Endurance glider deployments
-AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
+AR69-01,Neil Armstrong,2022-06-20T20:00:00,2022-07-16T08:35:00,Irminger 9
 EK20220629,Elakha,2022-06-29T15:18:00,2022-06-29T21:44:00,Endurance CSPP turn
 ZP20220702,Zephyr,2022-07-02T11:00:00,2022-07-04T23:00:00,Papa 9a Glider Deployments
 EK20220708,Elakha,2022-07-08T15:11:00,2022-07-08T22:03:00,Endurance CSPP turn and recovery
+TS20220714,Tenacious II,2022-07-14T02:00:00,2022-07-15T04:00:00,Pioneer 18c glider deployments
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:293]: Looks good

**Review by cdobs — APPROVED** [review]

**Review by reedan88 — APPROVED** [review]
Looks good.

**Comment by rtravis-whoi** [comment]
[Irminger 9 AM Updates [DT-160]](https://trello.com/c/YNyU7qml/160-irminger-9-am-updates-dt-160)

**Pull request merged**